### PR TITLE
Add orders to account

### PR DESCRIPTION
**Pull request by Svitlana-G**

### CODE REVIEW

File: frontend/src/components/account/Account/Account.module.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.gridTwoBlocks {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 20px;
+}
+
+.rightBlock {
+  background-color: white;
```
Comment (annaBalabak) [inline frontend/src/components/account/Account/Account.module.scss:15]: use variable

File: frontend/src/components/account/Orders/components/RecipientDetails/index. js [hunk 2]
```diff
@@ -0,0 +1 @@
+export {default} from './RecipientDetails'
```
Comment (annaBalabak) [inline frontend/src/components/account/Orders/components/RecipientDetails/index. js:1]: Correct mistake with a folder

File: frontend/src/components/account/SideMenu/SideMenu.module.scss [hunk 3]
```diff
@@ -3,7 +3,6 @@
 .menuAccount {
   background-color: white;
```
Comment (annaBalabak) [inline frontend/src/components/account/SideMenu/SideMenu.module.scss:4]: use variable

File: frontend/src/components/account/Orders/Orders.jsx [hunk 4]
```diff
@@ -1,7 +1,208 @@
...
+      <div>
+        <h2 className={styles.title}> Мої замовлення</h2>
+        <div className={styles.gridTable}>
+          <div>Номер</div>
+          <div>Дата</div>
+          <div>Отримувач</div>
+          <div>Сума</div>
+          <div>Статус</div>
+          <div></div>
+        </div>
```
Comment (annaBalabak) [inline frontend/src/components/account/Orders/Orders.jsx:111]: Add horizontal line after this block

File: frontend/src/components/account/Orders/Orders.jsx [hunk 5]
```diff
@@ -1,7 +1,208 @@
...
+                  )}
+                </button>
+              </div>
+            </div>
+          ))}
+
+        {!data && (
+          <div className={styles.noOrders}>
+            <p>У вас поки що немає замовлень.</p>
+            <img src="/images/no_orders.jpg" alt="" />
```
Comment (annaBalabak) [inline frontend/src/components/account/Orders/Orders.jsx:200]: Add alt text

File: frontend/src/components/account/Orders/Orders.module.scss [hunk 6]
```diff
@@ -0,0 +1,71 @@
+@import '../../../variables';
+
+.title {
+  font-size: 20px;
+  font-weight: 400;
```
Comment (annaBalabak) [inline frontend/src/components/account/Orders/Orders.module.scss:5]: It is a default font-weight

File: frontend/src/components/account/Account/Account.module.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
+.accountTitle {
+  font-size: 20px;
+  font-weight: 400;
```
Comment (annaBalabak) [inline frontend/src/components/account/Account/Account.module.scss:3]: It is a default font-weight

File: frontend/src/components/account/ChangePassword/ChangePassword.module.scss [hunk 8]
```diff
@@ -0,0 +1,5 @@
+.title {
+  font-size: 20px;
+  font-weight: 400;
```
Comment (annaBalabak) [inline frontend/src/components/account/ChangePassword/ChangePassword.module.scss:3]: It is a default font-weight

File: frontend/src/components/account/ContactDetails/ContactDetails.module.scss [hunk 9]
```diff
@@ -1,27 +1,15 @@
 @import '../../../variables';
 
+.block {
+  padding-left: 40px;
+}
+
 .accountTitle {
   font-size: 20px;
   font-weight: 400;
```
Comment (annaBalabak) [inline frontend/src/components/account/ContactDetails/ContactDetails.module.scss:9]: It is a default font-weight

File: frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+    <>
+      <div className={styles.flexData}>
+        <div>
+          <div className={styles.address}>
+            <div>Доставка:</div>
+            <div>
+              <span>місто</span> {city}
+            </div>
+            <div>
+              <span>вул.</span> {street} <span>буд.</span> {house}{' '}
```
Comment (annaBalabak) [inline frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx:23]: Do we need space here?

File: frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+              <span>кв.</span> {apartment}
+            </div>
+          </div>
+          <div className={styles.recipient}>
+            <div>Отримувач:</div>
+            <div>{recipient}</div>
+            <div>
+              <span>Тел.</span> {phone}{' '}
```
Comment (annaBalabak) [inline frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx:31]: Do we need space here?

File: frontend/src/components/account/SideMenu/SideMenu.module.scss
Comment (annaBalabak) [inline frontend/src/components/account/SideMenu/SideMenu.module.scss:1]: Line 25 Use a variable

File: frontend/src/components/account/Account/Account.module.scss (hunk 1 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Account/Account.module.scss:15]: done

File: frontend/src/components/account/SideMenu/SideMenu.module.scss (hunk 3 above)
Comment (Svitlana-G) [reply frontend/src/components/account/SideMenu/SideMenu.module.scss:4]: Done

File: frontend/src/components/account/ChangePassword/ChangePassword.module.scss (hunk 8 above)
Comment (Svitlana-G) [reply frontend/src/components/account/ChangePassword/ChangePassword.module.scss:3]: This style for h2, therefore I need font-weight

File: frontend/src/components/account/ContactDetails/ContactDetails.module.scss (hunk 9 above)
Comment (Svitlana-G) [reply frontend/src/components/account/ContactDetails/ContactDetails.module.scss:9]: This style for h2, therefore I need font-weight

File: frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx (hunk 11 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx:31]: No, deleted

File: frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx (hunk 10 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/components/RecipientDetails/RecipientDetails.jsx:23]: Yes

File: frontend/src/components/account/SideMenu/SideMenu.module.scss
Comment (Svitlana-G) [reply frontend/src/components/account/SideMenu/SideMenu.module.scss:1]: Done

File: frontend/src/components/account/Orders/Orders.jsx (hunk 5 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/Orders.jsx:200]: Added

File: frontend/src/components/account/Orders/components/RecipientDetails/index. js (hunk 2 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/components/RecipientDetails/index. js:1]: Add ; and rename index. js to index.js

File: frontend/src/components/account/Orders/Orders.jsx (hunk 4 above)
Comment (Svitlana-G) [reply frontend/src/components/account/Orders/Orders.jsx:111]: Added

**Pull request merged**